projects/static/projects/css/projects.css

```diff
@@ -1,2 +1,4 @@
-.allocation{border-bottom:1px solid #eee;margin-top:22px;margin-bottom:22px}.allocation:last-child{border-bottom:none}.allocation-inactive{display:none}.label-pending{background-color:#F0AD4E}.label-approved,.label-active{background-color:#74A93C}
+/*.allocation{border-bottom:1px solid #eee;margin-top:22px;margin-bottom:22px}.allocation:last-child{border-bottom:none}*/
```

Comment:
How is this generated?

Reply:
manually :worried:

Reply:
:) -- it may be a simple thing to install a formatter so it looks a bit nicer, I assumed it was output by a tool!